BetterStats/BetterStats.cs

```diff
@@ -568,8 +568,11 @@ public static void UIProductEntry__OnUpdate_Postfix(UIProductEntry __instance)
         [HarmonyPrefix, HarmonyPatch(typeof(UIStatisticsWindow), "ComputeDisplayEntries")]
         public static void UIProductionStatWindow_ComputeDisplayEntries_Prefix(UIStatisticsWindow __instance)
         {
+            if (Time.frameCount % 10 != 0)
```

Comment:
I added this because I was seeing a frame rate drop when the stat window was open. It doesn't seem like we really need to recompute this stuff as often as it gets called.